lib/docs/assets/css/main.css

```diff
@@ -492,6 +492,8 @@ ul.tsd-descriptions > li > :last-child > :last-child > :last-child {
   padding-bottom: 200px; }
 
 .row {
+  display: -ms-flexbox;
```

Comment:
The docs folder is auto generated during the build, these changes are because of an update typedoc component or it's sub component